### PR TITLE
[jabber] Fixed jingle build

### PR DESCRIPTION
**Pull request by nico-izo**

### CODE REVIEW

File: src/plugins/protocols/jabber/jabber.qbs [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
 
 Protocol {
+    property bool jingle: false
+
     Depends { name: "jreen" }
 
     cpp.includePaths: ["src"]
+    cpp.defines: {
+        var basicDefs = base
+        if(jingle)
```
Comment (euroelessar) [inline src/plugins/protocols/jabber/jabber.qbs]: `if (jingle)`

File: src/plugins/protocols/jabber/src/protocol/modules/jinglesupport/jinglesupport.cpp [hunk 2]
```diff
@@ -33,8 +32,8 @@
 #include <qutim/servicemanager.h>
 #include <qutim/icon.h>
 #include <qutim/menucontroller.h>
-#include <QAudioInput>
-#include <QAudioOutput>
+#include <QtMultimedia/QAudioOutput>
```
Comment (euroelessar) [inline src/plugins/protocols/jabber/src/protocol/modules/jinglesupport/jinglesupport.cpp]: Is it necessary?

File: src/plugins/protocols/jabber/src/protocol/modules/jinglesupport/jinglesupport.cpp [hunk 3]
```diff
@@ -165,7 +164,17 @@ void init_button(JingleButton *button)
 	MenuController::addAction<JContactResource>(button);
 }
 
-Q_GLOBAL_STATIC_WITH_INITIALIZER(JingleButton, button, init_button(x.data()))
+//Q_GLOBAL_STATIC_WITH_INITIALIZER(JingleButton, button, init_button(x.data()))
```
Comment (euroelessar) [inline src/plugins/protocols/jabber/src/protocol/modules/jinglesupport/jinglesupport.cpp]: Remove

File: src/plugins/protocols/jabber/src/protocol/modules/jinglesupport/jinglesupport.cpp [hunk 4]
```diff
@@ -165,7 +164,17 @@ void init_button(JingleButton *button)
...
+
+JingleButton *button()
```
Comment (euroelessar) [inline src/plugins/protocols/jabber/src/protocol/modules/jinglesupport/jinglesupport.cpp]: May you consider creating button in JingleSupport's constructor and deleting in JingleSupport's destructor or do something like this?

File: src/plugins/protocols/jabber/src/protocol/modules/jinglesupport/jinglesupport.cpp (hunk 2 above)
Comment (nico-izo) [inline src/plugins/protocols/jabber/src/protocol/modules/jinglesupport/jinglesupport.cpp]: It is necessary for me, without prefix I got a compile error.

**Comment by euroelessar** [comment]
Okay, as it's opt-in I'll merge it

**Pull request merged**